.github/workflows/performancetest.yml

```diff
@@ -0,0 +1,40 @@
+name: Performance-Test
+
+on:  
+  push:
+    branches:
+      - development
+
+jobs:
+  slsart-perf:
+    name: slsart Performance Test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [12.x]
+    env:
+      NPM_CONFIG_PREFIX: "~/.npm-global"
+      STAGE_NAME: merge-perf-test
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node }}
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-west-2
+      - run: npm install -g serverless@1.38.0
+      - run: npm install -g serverless-artillery
```

Comment:
Did you look into the run syntax
```
run: |
  npm install -g serverless
  npm install -g serverless-artillery 
```

or 

`run: npm install -g serverless serverless-artillery`

Reply:
I misunderstood how that | worked, I will make this change in a couple places now. Thanks!